aiconfig-docs/docs/getting-started.md

```diff
@@ -58,15 +58,31 @@ $ yarn add aiconfig
 
 ## Setup your OpenAI API Key
 
-For this tutorial, you will need to have an OpenAI key that has access to GPT-4.
+You will need to specifiy API keys for the model providers (i.e. OpenAI, Google, HuggingFace) you plan to use. We recommend adding your API keys as environment variables so that they are accessible for all projects. The python library will automatically detect and use them without you having to write any code.
```

Comment:
See comments I left to Andrew's PR in 

- https://github.com/lastmile-ai/aiconfig/pull/876#discussion_r1448458863
- https://github.com/lastmile-ai/aiconfig/pull/876#discussion_r1448459289